Configuration/PyReleaseValidation/python/relval_2017.py

```diff
@@ -55,7 +55,7 @@
            10824.501,
            10824.511,
            10824.521,
-           11624.911, 11642.911,
+           11634.911,11624.911, 11642.911,
```

Comment:
@silviodonato , @cvuosalo could you clarify why we need 13TeV for Run3 scenario? Shall we move everything about DD4HEP to 14TeV (also ZMM)?